crates/noirc_evaluator/src/ssa/context.rs

```diff
@@ -138,7 +151,7 @@ impl SsaContext {
     }
 
     //Display an object for debugging puposes
-    fn node_to_string(&self, id: NodeId) -> String {
+    fn id_to_string(&self, id: NodeId) -> String {
```

Comment:
`node_to_string` prints an entire instruction, e.g. `4: add (2) (3)`. 
`id_to_string` is responsible for printing just the numbers `(2)` and `(3)` without recurring and printing the whole instruction. Otherwise when printing `4: add (2) (3)` we'd be re-printing the entirety of ids 2 and 3 which would then recursively re-print their arguments and so on.

Reply:
ah I missed that thanks for clarifying